ansible/roles/jicofo/vars/main.yml

```diff
@@ -2,5 +2,7 @@
 jicofo_hostname: "{{ prosody_domain_name }}"
 jicofo_auth_domain: "auth.{{ prosody_domain_name }}"
 jicofo_auth_user: focus
-jicofo_auth_password: "{{ prosody_focus_user_secret }}"
+jicofo_auth_password: "{{ secrets_jicofo_focus }}"
```

Comment:
We should set 'replaceme' defaults for these secrets_ variables so that we do not fail if they aren't defined (such as at build time).